lib/cisco_node_utils/interface.rb

```diff
@@ -933,7 +933,11 @@ def default_switchport_mode
     end
 
     def switchport_trunk_allowed_vlan
-      config_get('interface', 'switchport_trunk_allowed_vlan', name: @name)
+      vlans = config_get('interface', 'switchport_trunk_allowed_vlan',
+                         name: @name)
+      vlans = vlans.join(',') if vlans.is_a?(Array)
+      vlans = Utils.normalize_range_array(vlans, :string) unless vlans == 'none'
```

Comment:
When would `vlans == 'none'` for this property?  The default value is it not `none` for this prop.